modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/TableViewSkinTest.java

```diff
@@ -46,19 +66,138 @@ public void test_JDK_8188164() {
         Scene scene = new Scene(tableView);
         scene.getStylesheets().add(TableViewSkinTest.class.getResource("TableViewSkinTest.css").toExternalForm());
 
-        Toolkit tk = Toolkit.getToolkit();
+        stageLoader = new StageLoader(scene);
+
+        Toolkit.getToolkit().firePulse();
+
+        TableHeaderRow header = (TableHeaderRow) tableView.lookup("TableHeaderRow");
+        assertEquals(100.0, header.getHeight(), 0.001, "Table Header height specified in CSS");
+    }
+
+    @Test
+    void testInitialColumnResizeNodePositions() {
+        TableView<String> tableView = new TableView<>();
```

Comment:
question: would it make sense to test with scene scale(s) other than 100%?  just to see if there is no failures with fractional scales?
(the asserts would need to use non-0 tolerance)

Reply:
It probably won't hurt. But since the mismatch is visible even at a normal scale, I didn't bothered to do so.

Reply:
... as a safeguard for any possible regressions?